### ISSUE TITLE
Numpy on MacOsX fails in our `CI` tests

### ISSUE THREAD

**Issue by opacam** [kivy/python-for-android#2087]
So a mac user will probably will not succeed building an `apk` with `numpy` in his requirements (or any recipe that depends on it...like `matplotlib` or any other one)

See also:
  - [this comment](https://github.com/kivy/python-for-android/pull/2077#issuecomment-593186149) at pr #2077
  - [this comment](https://github.com/kivy/python-for-android/pull/2073#issuecomment-593126362)  at pr #2073

**Comment by mainakbiswas** [comment]
Faced this same issue today.

created a main.py as below

```
import sys
print ('sys.version is', sys.version)
print('sys.path is', sys.path)

import numpy as np
a = np.arange(15).reshape(3, 5)
print('numpy seems to work:\n', a)
```

And then ran 
`p4a apk --private `pwd` --package=de.plapadoo.p4anumpy --name "p4a numpy" --bootstrap=sdl2 --version 0.1 --requirements=numpy --arch arm64-v8a`

The compilation failed.

**Comment by misl6** [comment]
Fixed via #2575 and #2586
